webui/react/src/types.ts

```diff
@@ -372,12 +376,14 @@ export interface Task {
 export interface ExperimentTask extends Task {
   archived: boolean;
   progress?: number;
+  resourcePool?: string;
```

Comment:
question in what cases do we have tasks or experiments that are missing the resourcepool tag?

Reply:
I was uncertain about this. With the current API, there should always be a resourcepool field. But if we use the current webui and point it to an older version of the master, the code will run fine as long as these fields are marked as optional. I'm not sure to what degree we care about the webui working with masters from previous patch versions. 

Should I change it to be a required field?